src/Raven.Client/Documents/Commands/StartTransactionsRecordingCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Net.Http;
+using Newtonsoft.Json;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Commands
+{
+    public class StartTransactionsRecordingCommand : RavenCommand
+    {
+        private readonly string _filePath;
+
+        public StartTransactionsRecordingCommand(string filePath)
+        {
+            _filePath = filePath;
+        }
+        public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
+        {
+            url = $"{node.Url}/databases/{node.Database}/start_transactions_recording?file_path={_filePath}";
```

Comment:
This should only be available to cluster admin, not any user

Reply:
done